solutions/instances/main.tf

```diff
@@ -96,6 +96,11 @@ module "resource_group" {
 # Observability Instance
 #######################################################################################################################
 
+locals {
+  parsed_existing_cloud_monitoring_crn = var.existing_cloud_monitoring_crn != null ? split(":", var.existing_cloud_monitoring_crn) : []
```

Comment:
why are we considering it as `[]` ?

Reply:
If the `existing_cloud_monitoring_crn` is null then the `parsed_existing_cloud_crn` will also be null.